scripts/install-env.sh

```diff
@@ -88,7 +90,14 @@ case $platform in
         ;;
 esac
 
-echo ""
-echo "${bold}install-env:${normal} Installing \`xargo\`."
-echo ""
-cargo install xargo
+echo "${bold}install-env:${normal} Checking if xargo is installed..."
+command -v xargo >/dev/null 2>&1
+if [[ $? -eq 1 ]]; then
+    echo "${bold}install-env:${normal} xargo is already installed."
+    exit 0
+else
+    echo "" 
+    echo "${bold}install-env:${normal} Installing \`xargo\`."
+    echo ""
+    cargo install xargo
```

Comment:
Similarly, can we run the install step with `set -e`? We'll need to `set +e` before running `command -v xargo` though.